test/unit-test-helpers/mock_ima.js

```diff
@@ -12,6 +12,7 @@ google =
       google.ima.linearAds = true;
       google.ima.delayAdRequest = false;
       google.ima.adsRenderingSettingsInstance = null;
+      google.ima.disableCustomPlaybackForIOS10Plus = false;
```

Comment:
Can you defined this up above as well? This way it does not start out as undefined.